### DISCUSSION TITLE
Add URLCheck to Accrescent

### DISCUSSION THREAD

**Discussion by Ilithy** [TrianguloY/URLCheck#181] [Ideas]
Hi;
It's about adding URLCheck to the application catalog of [Accrescent](https://github.com/accrescent/accrescent), a new application store focused on security and privacy.

Accrescent is currently in alpha, but already works well (even if its interface remains very basic, like the number of functions), but already has several advantages compared to Fdroid:
* Accrescent is much more secure.
* Accrescent also allows users to receive updates (on average) several days before Fdroid releases them.

[Accrescent Developer Console](https://github.com/accrescent/devconsole)
[Accrescent website](https://accrescent.app/)

Thank you so much

**Comment by TrianguloY** [comment]
Hmm, it requires manual generation and uploading, which I try to avoid (the play store also requires manual uploading, but at least I'm doing progress automating it). I'll have it on mind if I can automate it somehow.

Thanks for the suggestion!

**Reply by matchboxbananasynergy** [reply]
Hey there! Just chiming in to say that making the bundle building process as well as update process is planned to be much more automated in the future, so perhaps you'll consider it when that is the case!

**Reply by TrianguloY** [reply]
That'd be wonderful! Keep me updated if possible.

I don't mind if I need to configure a few things at first, but I would like it to build automatically after that (like f-droid).

**Reply by matchboxbananasynergy** [reply]
Understandable! Accrescent isn't like F-Droid in that it doesn't build apps itself, but rather developers build and sign their apps and upload them to Accrescent. That, however, doesn't mean the process can't be mostly automated. :)

There is already this https://github.com/accrescent/bundletool-gradle-plugin which is meant to largely automate building the app bundle that needs to be uploaded.

Accrescent's developer console is currently being re-written, and part of that is making it so those bundles can be pushed to it in an automated fashion, instead of you having to go to it to upload every new update.

Since you seem to be open to it/interested, I'll definitely try to keep you up-to-date!

P.S. I'm not the person behind Accrescent - I just think it's cool and try to help out where I can. :)

**Reply by TrianguloY** [reply]
Just want to quickly mention that I finally added an action to generate and upload an apk. It's signed with a different certificate that the play store, but still one I made: https://github.com/TrianguloY/UrlChecker/releases/

That should help while uploading the app to other stores, I think.
